src/main/java/co/org/osso/api/CalculatorService.java

```diff
@@ -0,0 +1,60 @@
+package co.org.osso.api;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class CalculatorService {
```

Comment:
I think the name `Calculator` is clear without `Service` in this case. The `FooService` pattern is certainly most common in web apps, but that's because most `@Service`s are centered around doing stuff with some form of data model (Book, User etc.) where as calculator... Not so much. It should be named `NumberService` if anything, but that's just silly. `Calculator` is perfect.